lib/Handlers/ifV2.ts

```diff
@@ -0,0 +1,48 @@
+import { NodeType } from '@voiceflow/general-types/';
+import { Node } from '@voiceflow/general-types/build/nodes/ifV2';
+
+import { HandlerFactory } from '@/lib/Handler';
+
+import { TurnType } from '../Constants/flags';
+import CodeHandler from './code';
+
+export type IfV2Options = {
+  endpoint?: string | null;
+  safe?: boolean;
+};
+
+const IfV2Handler: HandlerFactory<Node, IfV2Options | void> = ({ endpoint, safe } = {}) => ({
+  canHandle: (node, runtime) => node.type === NodeType.IF_V2 && !(runtime.turn.get<string[]>(TurnType.STOP_TYPES) || []).includes(NodeType.IF_V2),
+  handle: async (node, runtime, variables, program) => {
+    let outputPortIndex = -1;
+    const setOutputPort = function(index: number) {
+      outputPortIndex = index;
+    };
+    const codeHandler = CodeHandler({ endpoint, callbacks: { setOutputPort }, safe });
+
+    let code = '';
+    for (let i = 0; i < node.payload.expressions.length; i++) {
+      code += `
+            if(${node.payload.expressions[i]}) {
```

Comment:
should we wrap each individual condition in a separate try catch?

Reply:
yea I thought about this too. On the other hand, maybe it messes up the user logic even more if say they want condition1 to take precedence, condition1 is malformed and say condition3 is met, so they keep going down path3 instead of one. You've got a better product context tho